### ISSUE TITLE
Core Group Entrance / Quit Strategy

### ISSUE THREAD

**Issue by jyhi** [UICHCC/HCC-Regulations#8]
Currently we lack regulations on people joining or quitting the Core group.

The Core group is centric, and have access and power to almost all resources of HCC, the most important of which is infrastructure access (e.g. server password). People got enrolled in the Core group have access to all of these, thus it is improper to let people be in and out.

Few months ago @billzhonggz proposed a Core group quit strategy to ensure people quitting the Core group do not leak critical information. I think, in addition, an entrance strategy is also needed to ensure all people in Core group have ability to manage HCC.

You are encouraged to share your ideas (especially current Core members).

**Comment by jyhi** [comment]
Partial conclusion in WeChat group:

- Personal wish comes first
- Start a poll in Core group, and at least **1/2 or 2/3** people should agree with the entrance / quit
  - @DRJ31 prefers 1/2

**Comment by ecwu** [comment]
Two-thirds of the core members need to participate in decision-making and support must be greater than 1/2 of the core members.

e.g.: If there are 12 peoples in the core. Then more than 8 peoples (2/3 of 12) should participate in the decision-making process. And supporter should not less than 6 peoples (1/2 of 12).

**Comment by jyhi** [comment]
@ecwu I do not get your idea quite well. Do you mean that 1/3 of Core can be free from decision-making?

**Comment by ecwu** [comment]
@lmy441900 All core members participate in decision-making is the ideal situation.

**Comment by billzhonggz** [comment]
Agree. **ALL** members of the core should **ALWAYS** participate in affairs of the association. For any critical issue that involves voting, I suggest a 1/2 pass strategy.

**Comment by mchobbylong** [comment]
Agree for the 1/2 pass strategy.
Maybe there's someone who happen to be too busy to participate in affairs. I think it is strongly recommended to participate all the time, but not a must. Situations beyond human controls should be considered.

**Comment by billzhonggz** [comment]
I mean the *participate* here is not limited to offline stuff. All online/offline activities can be counted, including discussions.

**Comment by jyhi** [comment]
@mchobbylong If you have difficulties compiling\* this regulation please ask for help.

\* **verb (used with object):** to put together (documents, selections, or other materials) in one book or work. https://www.dictionary.com/browse/compile
